tests/console/yast2_proxy.pm

```diff
@@ -88,14 +88,15 @@ sub run {
 
     # check network interfaces with open port in firewall
     # repeat action as sometimes keys are not triggering action on leap if workers are slow
-    send_key_until_needlematch 'yast2_proxy_network_interfaces', 'alt-d', 2, 5;
-
-    wait_still_screen 1;
-    send_key 'alt-n';
-    wait_still_screen 1;
-    send_key 'alt-a';
-    wait_screen_change { send_key 'alt-o' };
-
+    if (!is_sle && !sle_version_at_least('15')) {
```

Comment:
Which distri are we aiming here? I believe we can use is_tumbleweed here

Reply:
will re-write this: using sub ti splitt into: setup_ldap, samba_current, samba_new so that to have a clear structure :) thanks for review!

Reply:
it should be:
if ((is_tumbleweed) || (is_sle && !sle_version_at_least('15'))